turbo.json

```diff
@@ -6,24 +6,12 @@
       "dependsOn": ["^build"],
       "outputs": ["dist/**"]  
     },
-    "test": {
-      "dependsOn": [],
-      "outputs": []
-    },
+    "test": {},
     "coverage": {
-      "dependsOn": [],
       "outputs": ["coverage/**"]
     },
-    "lint": {
-      "dependsOn": ["^lint"],
-      "outputs": []
-    },
-    "typescript": {
-      "dependsOn": ["^typescript"],
-      "outputs": []
-    },
-    "extract-translations": {
-      "outputs": []
-    }
+    "lint": {},
+    "typescript": {},
```

Comment:
I think this is mostly fine, but there is good reason to have typescript in dependency order, since a downstream package might depend on upstream type changes.